app/src/main/java/org/mozilla/fenix/library/bookmarks/SignInView.kt

```diff
@@ -22,7 +22,7 @@ class SignInView(
         get() = container
 
     val view: MaterialButton = LayoutInflater.from(container.context)
-        .inflate(R.layout.component_sign_in, container, true)
+        .inflate(R.layout.component_bookmark, container, true)
```

Comment:
I think doing this would have us inflating the bookmark layout twice. Why don't we remove the entire `SignInView` and move that logic into the `BookmarkView` (I think it's just 1 click listener)

Reply:
@ekager Done :)